pkg/ipamd/datastore/data_store_test.go

```diff
@@ -491,6 +932,58 @@ func TestGetIPStatsV4(t *testing.T) {
 	)
 }
 
+func TestGetIPStatsV4WithPD(t *testing.T) {
```

Comment:
nit: can this call `TestGetIPStatsV4` with PD boolean to simplify testing?

Reply:
Same as above, to avoid conditional checks.